demo/src/main/java/com/google/maps/android/utils/demo/MultiLayerDemoActivity.java

```diff
@@ -128,7 +128,7 @@ public void onFeatureClick(Feature feature) {
         KmlLayer kmlPolygonLayer;
         try {
             // KML Polyline
-            kmlPolylineLayer = new KmlLayer(getMap(), R.raw.south_london_line_kml, this, markerManager, polygonManager, polylineManager, groundOverlayManager);
+            kmlPolylineLayer = new KmlLayer(getMap(), R.raw.south_london_line_kml, this, markerManager, polygonManager, polylineManager, groundOverlayManager, null);
```

Comment:
Suggestion - in this or KmlDemoActivity, include an example of passing in the cache

Reply:
Ah, good point. I'll update `KmlDemoActivity` and reuse the `RetainFragment` + `ImagesCache` strategy there